public_html/wp-content/themes/pattern-directory/single-wporg-pattern.php

```diff
@@ -44,7 +62,12 @@
 							}
 							?>
 						</div>
-						<div class="pattern__report">
+						<div id="pattern-report"
+							class="pattern__report"
+							data-post-id="<?php echo intval( get_the_ID() ); ?>"
+							data-logged-in="<?php echo intval( is_user_logged_in() ); ?>"
+							data-user-has-reported="<?php echo intval( $user_has_reported ); ?>"
+							">
 							<button class="button">Report this pattern</button>
```

Comment:
For some reason this is showing as "You've reported this pattern" even when `data-user-has-reported` is set to `0`.

Reply:
Of course it is, because `Boolean( "0" )` is equal to `true`.

_.....Arrested development theme song starts playing...._

:) Thanks for the catch.